web/themes/custom/custom_sub_theme/scss/base/_importer.scss

```diff
@@ -0,0 +1 @@
+@forward "./general";
```

Comment:
Why would you forward general in importer and where are variables mixins partials ?

Reply:
there is no need of variables mixins and partials as i have used bootstrap classes in html , so html elements are automatically styled